backend/src/ai/router.py

```
@@ -0,0 +1,8 @@
+from fastapi import APIRouter
+from ..main.utils import verify_key
+
+router = APIRouter(prefix='/chats')
+
+@router.get('/')
+async def get_chats():
+    pass
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)